Formula/cbmc.rb

```diff
@@ -0,0 +1,53 @@
+class Cbmc < Formula
+  desc "CBMC: The C Bounded Model Checker"
+  homepage "https://www.cprover.org/cbmc/"
+  url "https://github.com/diffblue/cbmc.git", :using => :git, :tag => "cbmc-5.12"
```

Comment:
please specify a tag and a revision here.

Reply:
Done.